libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -176,6 +194,11 @@ class YulUtilFunctions
 	/// signature: (slot, [offset,] value)
 	std::string updateStorageValueFunction(Type const& _type, boost::optional<unsigned> const _offset = boost::optional<unsigned>());
 
+	/// Returns the name of a function will write the given value to
+	/// the specified address.
```

Comment:
Is the type the target or the source type?

Reply:
I would say it's both?

Reply:
Then what about calling this `writeToMemory`?

Reply:
I tried to use the existing naming pattern so I derived this name from `updateStorageValueFunction`. 
Maybe this doesn't apply well here, with storage and memory being rather different in how they work.

Reply:
Yes, for storage it makes more sense to call it update, because for packed types, it has to read first, update and then write, but for memory it is just a simple write.